test/test.taskbatching.js

```diff
@@ -0,0 +1,156 @@
+/* global beforeEach:true, describe:true, expect:true, it:true, Modernizr:true */
+describe('Task Batching', function() {
+    'use strict';
+
+    beforeEach(function(done) {
+        if (Modernizr.indexeddb) {
+            localforage.setDriver(localforage.INDEXEDDB, function() {
+                window._batches = [];  // FIXME: Is there a better place to store this stub recording?
```

Comment:
Is this only stored for the test? If not, I would think just attaching it to the localForage instance would be best, either way. eg:

```javascript
var localforage = LocalForage.createInstance();
console.log(localforage._batch);
# => []
```

It shouldn't be a "public" variable or part of the API but I think that would work.